src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -88,6 +105,60 @@ public HttpResponse<String> sendNewSessionRequest(String sessionRequestBody)
         return sendHttpRequest(request);
     }
 
+    public HttpResponse<String> sendEventRequest(String sessionId)
+            throws IOException, InterruptedException {
+
+        URI eventsEndpointURI =
+                URI.create(
+                        "https://"
+                                + STACK_NAME
+                                + ".review-a."
```

Comment:
It was in common lib so that I can use the same request for both KBV and Address, when KBV ticket comes it'll be a case of updating the url to use either `-a` or `-k` depending on where the tests are being ran. But maybe its more simpler to just duplicate the requests across both CRIs

Reply:
We should instead use the `TestHarnessExecuteUrl` output from the test resources stack - it already selects the custom domain based on the CRI and we don't need to duplicate the logic here